src/npc/picolyteBottle.c

```diff
@@ -124,10 +204,21 @@ void sub_0806E23C(void) {
     }
 }
 
-void sub_0806E250(Entity* this, ScriptExecutionContext* context) {
+void sub_0806E250(PicolyteBottleEntity* this, ScriptExecutionContext* context) {
     context->condition = 0;
     if (CheckKinstoneFused(0x31) != 0) {
         context->condition = 1;
     }
     gActiveScriptInfo.flags |= 1;
 }
+
+void (*const PicolyteBottle_Actions[])(PicolyteBottleEntity*) = {
+    PicolyteBottle_Init,
+    PicolyteBottle_Action1,
+    PicolyteBottle_Action2,
+    nullsub_111,
+};
+
+const u8 gUnk_081142CC[] = { 1, 0, 0, 1, 0, 1, 0, 1, 1, 0, 1, 0, 0, 1, 1, 0 };
+const u8 gUnk_081142DC[] = { 1, 0, 0, 0, 0, 0, 1, 0, 1, 0, 0, 0, 0, 1, 0, 0,
+                             1, 0, 0, 0, 0, 0, 1, 0, 0, 1, 0, 0, 0, 0, 1, 0 };
```

Comment:
I think we decided to put data like this, that gets used only within a single function, as a `static const` inside the function, right?

Reply:
I don't see other people do it nor the discussion back in September really reached a decision.

Reply:
I started doing it, but yeah, we never reached a final decision

Reply:
I've been doing it. It's what they originally did and I think it makes sense for stuff that is only accessed once. Stuff like animation frames can reside outside of the functions though. Everything else (random offsets, function pointer tables, etc) I've been putting in their respective function.

I haven't made a huge deal about it since it can be moved around later.